bcgov/HOUS-permit-portal
# adding historic site integration and geometry intersect

## Description

This adds the historic site custom compliance.  It will check off something if it is found in the historic site search.  We may may want to separate provincial sites later.

## What type of PR is this? (check all applicable)

- [ x ] 🍕 Feature
- [ x ] ✅ Test

## Related Tickets & Documents

Split off from story https://hous-bssb.atlassian.net/browse/BPHH-177 into
https://hous-bssb.atlassian.net/browse/BPHH-603

## Steps to QA

Test specs pass.
When you run save on the permit application, it should trigger the backend to call the address and return the historic site.  Please read the specs to test sites that are historic.
This also requires customCompliance module "HistoricSite" to be set up on a requirement.

## UI Accessibility Checklist
No front end here.

## General Checklist

- [ x] ✅ Provide tests for your changes where relevant
- [ x] 📝 Use descriptive commit messages
- [ x] 📗 Update any related documentation and include any relevant screenshots
- [ x] 🗂️ Move your relevant story/task to the _Ready for Code Review_ state
